pigar/parser.py

```diff
@@ -55,13 +56,22 @@ def parse_imports(package_root, ignores=None):
     return imported_modules, user_modules
 
 
+def _filter_ipynb_commands(cell_lines):
+    """
+    Filter out cells with magics and shell commands
+    e.g %matplotlib inline or !pip install pigar
+    """
+    return re.sub(r"^((\s)*(!|%).*$)|(\n\s*%.*)", "", cell_lines)
```

Comment:
Maybe we should handle [`var = !command`](https://ipython.org/ipython-doc/3/interactive/shell.html#string-lists) as well.